iotdb/src/main/java/org/apache/iotdb/db/engine/querycontext/ReadOnlyMemChunk.java

```diff
@@ -89,6 +89,8 @@ private void init() {
           break;
       }
     }
+    //release memory
+    memSeries = null;
```

Comment:
Would this cause resource leak? It is better to have something like 
`
memSeries.release()
memSeries = null
`

Reply:
1. no such method...
2. memSeries is invisible for users and after this method (`init()`), it is never used.